platform-api/src/routers/forms.py

```
@@ -0,0 +1,136 @@
+    await notify_discord(body, settings)
+
+    return SubmissionResponse(id=doc_id)
+
+@router.get("/submissions", response_model=list[SubmissionListItem])
+async def list_submissions(
+    site: str = Query("capstone", description="Target site/workspace"),
+    status: str = Query("submitted", description="Submission status filter (use 'all' for no filter)"),
+    limit: int = Query(20, ge=1, le=100, description="Max records to return"),
+    offset: int = Query(0, ge=0, description="Number of records to skip for pagination"),
+    sanity: SanityClient = Depends(get_sanity),
+    _: bool = Depends(verify_admin_api_key)  # Protect route
```

Ruff (0.15.11)

[warning] Boolean-typed positional argument in function definition

(FBT001)